doc/languages.rst

```diff
@@ -681,6 +681,14 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       .. syntax-checker-config-file:: flycheck-perlcriticrc
 
+.. supported-language:: Nix
```

Comment:
Please keep the list of languages alphabetically sorted—"Nix" should go before "Markdown".